package.json

```diff
@@ -19,7 +19,7 @@
   "module": "dist/custom-elements/index.js",
```

Comment:
Have you checked whether this is still the way to go? Cause I assume, this file no longer exists.